packages/wp-story-editor/src/components/fontCheck/index.js

```diff
@@ -58,6 +62,22 @@ export const FontCheck = () => {
       const inUseFonts = getInUseFontsForPages(storyPages);
       const allFonts = await getFonts({ include: inUseFonts.join(',') });
 
+      if (!fontsChecked) {
```

Comment:
Hmm why was this new state variable needed? Why did it fire too often before?

Can we accomplish the same by changing the deps array below to `[isStoryLoaded]`\*? (and ignore lint warnings)

\* or maybe to `[isStoryLoaded, updateElementsByFontFamily]`

Reply:
I think the storyPages dependancy ends up cycling so yes I think removing it from the array will fix this

Maybe from the this part of the `updateElementsByFontFamily`...
```
const updatedPages = state.pages.map((page) => {
    const updatedElements = page.elements.map((element) => {
      if (element?.font?.family === family) {
        return updateElementWithUpdater(element, propertiesOrUpdater);
      }
      return element;
    });

    return {
      ...page,
      elements: updatedElements,
    };
  });
```